addon/Feeds/TopSitesFeed.js

```diff
@@ -25,7 +26,9 @@ module.exports = class TopSitesFeed extends Feed {
       if (experiments.screenshots) {
         try {
           links = yield getScreenshots(links, site => {
-            if (site.favicon_height >= 64 && site.favicon_width >= 64) {
+            if (site.favicon_height >= 64 && site.favicon_width >= 64 && ["/", ""].includes(urlParse(site.url).pathname)) {
```

Comment:
I was actually trying to optimize so that `urlParse()` isn't called if one of the previous two conditions fails. So, maybe I'll just wrap that up in a function like `isRootDomain()` instead for readability.